docs/remote_inputs.md

```diff
@@ -0,0 +1,156 @@
+# Overview of remote nCoV files (intermediate build assets)
+
+We have two repos which routinely upload files to s3 buckets: `ncov-ingest` and `ncov`.
+Each of those runs separate pipelines for `gisaid` and `genbank` (aka `open`) data sources, with the intention that the latter are available for public download.
+These files are intended to be available for use as starting points for running the `ncov` pipeline.
+This page details where different assets come from, and how they may be used as inputs for your own workflow.
+
+The GISAID data is stored at `s3://nextstrain-ncov-private` and the open (GenBank) data on the public bucket `s3://nextstrain-data/files/ncov/open`
```

Comment:
```suggestion
The GISAID data is stored at `s3://nextstrain-ncov-private` and is not publicly available, in line with the GISAID Terms of Use (this is used internally by Nextstrain).  The open (GenBank) data is available in the public bucket `s3://nextstrain-data/files/ncov/open`
```

I just think it might be good to be extra-explicitly-clear that this data is not available.